src/generated/resources/data/techarium/recipes/exchange_station/dirt.json

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "techarium:exchange_station",
+  "itemIn": {
+    "id": "minecraft:gold_ingot",
```

Comment:
I think it might just be better to hardcode this to gold  and just pass in a number here, cause we're just gonna be drawing a little G or smth to represent gold and not the actual item

Reply:
nvm